api/src/main/java/org/apache/cloudstack/api/response/CapabilitiesResponse.java

```diff
@@ -124,6 +124,14 @@ public class CapabilitiesResponse extends BaseResponse {
     @Param(description = "the retention time for Instances disks stats", since = "4.18.0")
     private Integer instancesDisksStatsRetentionTime;
 
+    @SerializedName("storagefsvmmincpucount")
```

Comment:
Can these constants be moved to ApiConstants class?

Reply:
done